nexus/external-api/output/nexus_tags.txt

```diff
@@ -170,6 +168,8 @@ ip_pool_view                             GET      /v1/system/ip-pools/{pool}
 API operations found with tag "system/metrics"
 OPERATION ID                             METHOD   URL PATH
 system_metric                            GET      /v1/system/metrics/{metric_name}
+system_timeseries_query                  POST     /v1/system/timeseries/query
+system_timeseries_schema_list            GET      /v1/system/timeseries/schema
```

Comment:
Originally posted on #6873, but that no longer has a schema list endpoint:

I think this should be `/schemas` instead of `/schema` since it's a list endpoint. Since we're breaking the API here anyway, it's as good a time there will ever be to fix it. @bnaecker ?